cfpb/consumerfinance.gov
# Tightens up TDP activities search frontend routing logic

## Testing
 - Pull and build
 - Navigate to http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/
 - Check and uncheck an item
 - You shouldn't end up with a stray `?q=`
